playbook/app/pb_kits/playbook/pb_star_rating/_star_rating.tsx

```diff
@@ -1,22 +1,20 @@
-/* @flow */
+import React from "react"
+import classnames from "classnames"
 
-import React from 'react'
-import classnames from 'classnames'
+import { buildAriaProps, buildDataProps } from "../utilities/props"
```

Comment:
A good thing to check for future PRs is that our globalProps is always included. This kit is so old that it didn't in fact have those. We'll update in a separate story, but I wanted you to be aware that we should always try to look for that as well in future PRs.